DependencyInjection/Configuration.php

```diff
@@ -45,8 +45,7 @@ public function getConfigTreeBuilder()
                             ->defaultValue(array('Default'))
                         ->end()
                         ->scalarNode('subject_provider')->defaultValue('mremi_contact.subject_provider.noop')->cannotBeEmpty()->end()
-                        ->booleanNode('captcha_disabled')->defaultFalse()->end()
-                        ->enumNode('captcha_type')->defaultValue('genemu_captcha')->values(array('genemu_captcha', 'genemu_recaptcha'))->end()
+                        ->scalarNode('captcha_type')->defaultFalse()->end()
```

Comment:
I think this is better, no mix between boolean and string and force the final user to provide a value if configured:

```php
->scalarNode('captcha_type')->defaultNull()->cannotBeEmpty()->end()
```